reV/bespoke/bespoke.py

```diff
@@ -962,16 +962,21 @@ def _parse_points(excl_fpath, res_fpath, tm_dset, resolution,
     def _check_files(self):
         """Do a preflight check on input files"""
 
-        if not os.path.exists(self._excl_fpath):
-            raise FileNotFoundError('Could not find required exclusions file: '
-                                    '{}'.format(self._excl_fpath))
-
-        with h5py.File(self._excl_fpath, 'r') as f:
-            if self._tm_dset not in f:
-                raise FileInputError('Could not find techmap dataset "{}" '
-                                     'in exclusions file: {}'
-                                     .format(self._tm_dset,
-                                             self._excl_fpath))
+        paths = self._excl_fpath
+        if isinstance(self._excl_fpath, str):
+            paths = [self._excl_fpath]
+
+        for path in paths:
+            if not os.path.exists(path):
+                raise FileNotFoundError(
+                    'Could not find required exclusions file: '
+                    '{}'.format(path))
+
+            with h5py.File(path, 'r') as f:
+                if self._tm_dset not in f:
+                    raise FileInputError('Could not find techmap dataset "{}" '
+                                         'in exclusions file: {}'
+                                         .format(self._tm_dset, path))
```

Comment:
Okay so i'm pretty sure that tm_dset only needs to be on ONE of the excl file paths. Can you try the following? This is how the exclusions should be handled. 

```
from reV.handlers.exclusions import ExclusionLayers
with ExclusionLayers(paths) as excl:
    if self._tm_dset not in excl:
        msg = '...'
        logger.error(msg)
        raise FileInputError(msg)

Reply:
Oh great point, this is just an artefact of me still learning the finer details. I will add that change!